webapp/lib/platform.dart

```diff
@@ -138,13 +138,13 @@ Future updateUnread(List<Conversation> conversations, bool newValue) async {
       : "${conversations.length} conversations"
   }");
   if (conversations.isEmpty) return null;
-  var batch = _firestoreInstance.batch();
+  var batch = FsDocUpdate(_firestoreInstance.batch());
   int batchSize = 0;
```

Comment:
Where does batch size get updated here? It seems to always be zero?

Reply:
Thanks! Good catch! Fixed.